packages/picasso/picasso.0.4.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "Ghiles Ziat <ghiles.ziat@epita.fr"
+authors: [
+  "Ghiles Ziat <ghiles.ziat@epita.fr"
+  "Matthieu Journault <Matthieu.journault@lip6.fr>"
+  ]
+homepage: "https://github.com/ghilesZ/picasso"
+bug-reports: "https://github.com/ghilesZ/picasso/issues"
+dev-repo: "git+https://github.com/ghilesZ/picasso"
+build: [
+  ["dune" "subst"] {pinned}
```

Comment:
```suggestion
  ["dune" "subst"] {dev}
```
That is from a bug in dune < 2.7, we can fix it manually here on the PRs, no action needed from your side